src/Microsoft.VisualStudio.Composition/RuntimeComposition.cs

```diff
@@ -62,6 +65,7 @@ private RuntimeComposition(IEnumerable<RuntimePart> parts, IReadOnlyDictionary<T
                 e => (IReadOnlyCollection<RuntimeExport>)e.ToImmutableArray());
         }
 
+        [MessagePackFormatter(typeof(MessagePackCollectionFormatter<RuntimePart>))]
         public IReadOnlyCollection<RuntimePart> Parts
```

Comment:
Is this an optmization or does MessagePack not handle collections internally? ie would this not work if RuntimePart has the formatter attribute on it?